data/crac-file/crac-impl/src/test/java/com/farao_community/farao/data/crac_impl/AbstractRemedialActionTest.java

```diff
@@ -0,0 +1,54 @@
+package com.farao_community.farao.data.crac_impl;
+
+import com.farao_community.farao.data.crac_api.UsageRule;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+/**
+ * @author Alexandre Montigny {@literal <alexandre.montigny at rte-france.com>}
+ */
+abstract public class AbstractRemedialActionTest {
+
+    public ArrayList<UsageRule> createUsageRules() {
+        UsageRule mockedUsageRule = Mockito.mock(UsageRule.class);
+        ArrayList<UsageRule> usageRules = new ArrayList<>();
+        usageRules.add(mockedUsageRule);
+        return usageRules;
+    }
```

Comment:
this method is not needed anymore if you write another simple constructor without usage rules

Reply:
I keep it as I need to test the different type of constructors